### PR TITLE
Update manage-admins.md

### PR DESCRIPTION
**Pull request by dagoltz**
document a limitation in overriding roles and add the "look-alike" AAD roles that are assignable as built-in roles.

### CODE REVIEW

**Review by dcurwin — APPROVED** [review]

**Comment by dcurwin** [comment]
@dagoltz  - Please enter `#sign-off` in a new comment box when you're ready to publish. Thanks!

File: CloudAppSecurityDocs/manage-admins.md [hunk 1]
```diff
@@ -73,6 +73,12 @@ By default, the following Office 365 and [Azure AD](/azure/active-directory/role
 
 The following specific admin roles can be configured in the Defender for Cloud Apps portal:
 
+- **Global administrator**: Has **Full access** in Defender for Cloud Apps similar to the Azure AD Global administrator role.
```
Comment (ShannonLeavitt) [inline CloudAppSecurityDocs/manage-admins.md:76]:
```suggestion
- **Global administrator**: Has **Full access** in Defender for Cloud Apps, similar to the Azure AD Global administrator role.
```
Minor edit

File: CloudAppSecurityDocs/manage-admins.md [hunk 2]
```diff
@@ -130,7 +136,7 @@ The following specific admin roles can be configured in the Defender for Cloud A
 
 ## Override admin permissions
 
-If you want to override an administrator's permission from Azure AD or Office 365, you can do so by manually adding the user to Defender for Cloud Apps and assigning the user permissions. For example, if you want to assign Stephanie, who is a Security reader in Azure AD to have **Full access** in Defender for Cloud Apps, you can add her manually to Defender for Cloud Apps and assign her **Full access** to override her role and allow her the necessary permissions in Defender for Cloud Apps.
+If you want to override an administrator's permission from Azure AD or Office 365, you can do so by manually adding the user to Defender for Cloud Apps and assigning the user permissions. For example, if you want to assign Stephanie, who is a Security reader in Azure AD to have **Full access** in Defender for Cloud Apps, you can add her manually to Defender for Cloud Apps and assign her **Full access** to override her role and allow her the necessary permissions in Defender for Cloud Apps. Note that it is not possible to override Azure AD roles who grant Full access (Global administrator, Security administrator and Cloud App Security administrator).
```
Comment (ShannonLeavitt) [inline CloudAppSecurityDocs/manage-admins.md:139]:
```suggestion
If you want to override an administrator's permission from Azure AD or Office 365, you can do so by manually adding the user to Defender for Cloud Apps and assigning the user permissions. For example, if you want to assign Stephanie, who is a Security reader in Azure AD to have **Full access** in Defender for Cloud Apps, you can add her manually to Defender for Cloud Apps and assign her **Full access** to override her role and allow her the necessary permissions in Defender for Cloud Apps. Note that it is not possible to override Azure AD roles that grant Full access (Global administrator, Security administrator and Cloud App Security administrator).
```
Minor grammar edit

**Comment by PRMerger5** [comment]
@ShannonLeavitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by dcurwin — APPROVED** [review]

**Comment by dagoltz** [comment]
#sign-off

**Pull request merged**